thecourseforum/theCourseForum2
# Make cursor pointer on FAQ dropdown

## GitHub Issues addressed

- This PR closes https://github.com/thecourseforum/theCourseForum2/issues/700

## What I did

- Added a CSS statement to change the cursor to a pointer

## Screenshots

- Before
<img width="1294" alt="Screenshot 2023-10-22 at 3 41 21 PM" src="https://github.com/thecourseforum/theCourseForum2/assets/53921230/e74d6ec5-0e9e-4043-b173-beaf6c99a055">

- After
<img width="1298" alt="Screenshot 2023-10-22 at 3 42 10 PM" src="https://github.com/thecourseforum/theCourseForum2/assets/53921230/47c66045-2275-4e4e-8c48-d48b80cbc3e4">

## Testing

1. Go to https://thecourseforum.com/
2. Scroll down to the FAQ section
3. Hover mouse over any of the dropdown items
4. See cursor change to a pointer

## Questions/Discussions/Notes

- The cursor will turn into a pointer even when it's to the right of the FAQ text rather than directly on the text (this is the case in the "after" screenshot as well). I think this is valid because the FAQ question is still clickable when the pointer is to the right of the question.
